WcaOnRails/config/locales/en.yml

```diff
@@ -1537,6 +1537,8 @@ en:
       past_from: "Past from %{year}"
       past_all: "Past from all years"
       no_delegates: "None"
+      delegates_loading: "Waiting to finish loading all delegates..."
+      no_delegates_found: "No delegate found."
```

Comment:
Maybe add plural like so: "No Delegate(s) found."?

Reply:
The default just says "No results found." so maybe "No Delegates found." is good?